lms/static/js/student_account/views/RegisterView.js

```diff
@@ -92,7 +94,8 @@
                     eventValue: 0,
                     transport: 'beacon'
                 });
-
+                fbq('track', 'CompleteRegistration', {status: 'failed'});
+                snaptr('track', 'SIGN_UP', {success: 0});
```

Comment:
I don't like that you're defining the script in the theme repo and use it here, let us talk.

Reply:
Or basically this needs to be controlled by configs

Reply:
I see, the way the theme is built I cannot run this from within the theme, and according to edX we should add the scripts for analytics in theme. However, we need to do a check here if these objects exist.